### PR TITLE
fauxreilly:0.1.1

### PR DESCRIPTION
**Pull request by dei-layborer**
I am submitting
- [ ] a new package
- [x] an update for a package

Description: Updates the ``fauxreilly`` package to fix how images are added, since it was broken before.

### CODE REVIEW

**Comment by dei-layborer** [comment]
I'm not clear on what's causing the issue here, so any help would be appreciated.

**Comment by jneug** [comment]
Seems like you forgot to commit the main `lib.typ` file.

**Comment by dei-layborer** [comment]
That is *amazing*.  It's been that kind of day/week/month/year....

File: packages/preview/fauxreilly/0.1.1/typst.toml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+entrypoint = "lib.typ"
+authors = ["Dei Layborer"]
+license = "GPL-3.0"
+compiler = "0.11.1"
+description = "A package for creating O'Rly- / O'Reilly-type cover pages"
+repository = "https://github.com/dei-layborer/fauxreilly"
+categories = ["components", "layout", "fun"]
+exclude = [
+	"$$$-gimmie.svg",
+	"made-with-(2s)-2,6-diamino-n-[(2s)-1-phenylpropan-2-yl]hexanamide-n-[(2s)-1-phenyl-2-propanyl]-l-lysinamide.svg",
```
Comment (elegaanz) [inline packages/preview/fauxreilly/0.1.1/typst.toml:13]: You should update the file name here.

```suggestion
	"made-with.svg",
```

**Comment by elegaanz** [comment]
Thanks!

**Pull request merged**